src/Stratis.FederatedPeg.Features.FederationGateway/TargetChain/WithdrawalExtractor.cs

```diff
@@ -71,7 +71,9 @@ private IWithdrawal ExtractWithdrawalFromTransaction(Transaction transaction, ui
 
         private bool IsTargetAddressCandidate(TxOut output)
         {
-            return output.ScriptPubKey.Hash.GetAddress(this.network) != this.multisigAddress && !output.ScriptPubKey.IsUnspendable;
+            // TODO: Currently implemented for new multisig. Revert for old multisig.
+            // return output.ScriptPubKey.Hash.GetAddress(this.network) != this.multisigAddress && !output.ScriptPubKey.IsUnspendable;
```

Comment:
Is this comment staying or going ? Confused at the TODO, then some commented out code.

Reply:
Removed.

Reply:
Cool, thanks.